sources/dynamodb/schema_test.go

```diff
@@ -209,6 +229,15 @@ func TestProcessSchema_FullDataTypes(t *testing.T) {
 				},
 			},
 		},
+		{
+			Table: &dynamodb.TableDescription{
+				TableName: &tableNameA,
+				KeySchema: []*dynamodb.KeySchemaElement{
+					{AttributeName: &attrNameA, KeyType: &hashKeyType},
+					{AttributeName: &attrNameB, KeyType: &sortKeyType},
+				},
+			},
+		},
```

Comment:
same as above. Needed because of separate DescribeTable call now for GetConstraints and GetIndexes.